hooks/terraform_validate.sh

```diff
@@ -43,16 +80,64 @@ function per_dir_hook_unique_part {
   local -a -r args=("$@")
 
   local exit_code
-  local validate_output
+  #
+  # Get hook settings
+  #
+  local retry_once_with_cleanup=false
+
+  IFS=";" read -r -a configs <<< "${HOOK_CONFIG[*]}"
+
+  for c in "${configs[@]}"; do
+
+    IFS="=" read -r -a config <<< "$c"
+    key=${config[0]}
+    value=${config[1]}
+
+    case $key in
+      --retry-once-with-cleanup)
+        retry_once_with_cleanup=$value
```

Comment:
Should we take first or last parameter occurance when there are more than one specified (e.g. by mistake)? For the former (first takes precedence) we should add something like this:
```suggestion
        if [ ! $retry_once_with_cleanup ]; then retry_once_with_cleanup=$value; fi
```

Reply:
Better stop execution and ask what user check configuration.


```suggestion
        if [ $retry_once_with_cleanup ]; then 
          common::colorify "yellow" 'Invalid hook config. You specify more than one "--retry-once-with-cleanup" flag"'
          exit 1
        fi
        retry_once_with_cleanup=$value
```

Reply:
Yep, makes sense!